pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "cdiserrors"
+version = "1.0.0"
+description = "Gen3 shared exceptions and utilities."
+authors = ["CTDS UChicago <cdis@uchicago.edu>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+cdislogging = "^1.0.0"
+Flask = {version = "^1.1.2", optional = true}
```

Comment:
might be nice to add a quick note to the README on how to install this library with/without flask